gcloud/bigquery/dataset.py

```diff
@@ -58,6 +85,29 @@ def path(self):
         return '/projects/%s/datasets/%s' % (self.project, self.name)
 
     @property
+    def access_grants(self):
+        """Dataset's access grants.
+
+        :rtype: list of :class:`AccessGrant`
+        :returns: roles granted to entities for this dataset
+        """
+        return list(self._access_roles)
```

Comment:
In the same vein as my comment on `TestAccessRoles`, this is probably meant to be `_access_grants`?